contactsummary/js/test/controllers/key-details.controller.spec.js

```diff
@@ -42,7 +42,7 @@ define([
       });
 
       it('subscribes for contract refresh event', function () {
```

Comment:
the description is not true anymore. why are we changing refresh to delete?

Reply:
The event event name `refresh-contract` was not following our naming convention, so is changed into `Contract::deleted`. Making it more readable and specific.

Reply:
Yeah, I understand about the convention. I am more concerned about the "refresh" and "delete" terms. Does that mean that the previous name was wrong and didn't tell what the actual function does?

Reply:
@igorpavlov 
> Does that mean that the previous name was wrong and didn't tell what the actual function does?

Not exactly! I am making it  only more specific so, that we will know the exact event we are subscribing to. For me `contract-refresh` seemed generic and hard to find when it is published by it's name.